commec/screen.py

```diff
@@ -231,6 +236,14 @@ def add_args(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         action="store_true",
         help="Re-use any pre-existing output for this Screen run (cannot be used with --force)",
     )
+    screen_logic_group.add_argument(
+        "-r",
+        "--regions",
+        dest="regions",
+        nargs="+",
+        default=[],
+        help="A list of countries or regions to add context to list compliance i.e. NZ US CH",
```

Comment:
could include more about valid region_group values for this (e.g. Australia Group) somewhere...